Simulator/Utils/Global.h

```diff
@@ -65,6 +64,7 @@ typedef unsigned long long int uint64_t;   //included in inttypes.h, which is no
    //#include "Norm.h"
    #include "Coordinate.h"
    #include "VectorMatrix.h"
+   #include "memory"
```

Comment:
Should this be `#include <memory>`?

Reply:
Fixed. It is odd that both include forms work and point to the same file.